test/neighborhood_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule GameOfLife.NeighborhoodTest do
+  use ExUnit.Case, async: true
+
+  setup _the_list do
+    {:ok, [
+            of_two_living_cells: [{1,1},{1,2}],
+            of_three_living_cells: [{1,1},{1,2},{2,2}],
```

Comment:
These are small beans, but each of the last three lists are only used by one method. Why do you include them in the context where they are available to all tests?

Reply:
Fact: It does not affect test speed, and were this module to be extended, these test cases could be reused.
Feelings: I think it makes the test read easier and expresses what's going on better than raw data would, and for a small module like this it's less cluttered than having multiple setup contexts.